tests/test_models/test_engine/test_db_storage.py

```diff
@@ -176,6 +176,22 @@ def test_key_format(self):
     @skipIf(storage_t != 'db', "not testing db storage")
     def test_storage_var_created(self):
         """ DBStorage object storage created """
-        from models.engine.db_storage import DBStorage
-        print(type(storage))
         self.assertEqual(type(storage), DBStorage)
+
+    def test_storage_get_method(self):
+        """tests FileStorage method to retrieve one object"""
+        # gets initial counts of all objects and all State objects in storage
+        count = storage.count()
+        count_state = storage.count(State)
+        # tests get method when id is not found
+        self.assertIs(storage.get(State, 89), None)
```

Comment:
We have a count of how many States we have... Why not make this dynamic to ensure that the id passed is higher than our number of states? Would help prevent this from failing just because there's a lot of States or something.

Reply:
My understanding is that the id is automatically generated by the primary key constraint, which will typically be several (10+) digit hex number.  With ids like "fad30659-74f7-40bd-bf9a-52f41580e316" there should be no valid id "89" no matter how many states we have, and this test of a bad id number does not need to be dynamic.  Can you think of an example when this wouldn't be the case?

Reply:
Don't mind me. I was tired and forgot we were using UUID strings and not just integers. Should be perfectly fine!